build.sbt

```diff
@@ -210,6 +222,26 @@ lazy val rholangCLI = (project in file("rholang-cli"))
   )
   .dependsOn(rholang)
 
+lazy val rholangProtoBuildJar = Def.task(
+  (assemblyOutputPath in (assembly)).value
+)
+lazy val _incrementalAssembly = Def.taskDyn(
+  if (jarOutDated((rholangProtoBuildJar).value, (Compile / scalaSource).value))
+    (assembly)
+  else 
+    rholangProtoBuildJar
+)
+lazy val incrementalAssembly = taskKey[File]("Only assemble if sources are newer than jar")
+lazy val rholangProtoBuild = (project in file("rholang-proto-build"))
+  .settings(commonSettings: _*)
+  .settings(
+    name := "rholang-proto-build",
+    incrementalAssembly in Compile := Def.sequential(
```

Comment:
Tell me about the use `Def.sequential` here.  My understanding is that it's intended to be used to run several tasks in sequence.

Reply:
Ah, yes it is. That is probably left over from when I was trying various thing to get it to work and used to have multiple tasks in there. Will remove the now unnecessary `sequential`.